src/usr/local/www/system.php

```diff
@@ -456,7 +456,8 @@ function is_timezone($elt) {
 	'Timezone',
 	$pconfig['timezone'],
 	array_combine($timezonelist, $timezonelist)
-))->setHelp('Select the timezone or location within the timezone to be used by this system.');
+))->setHelp('Select the timezone or location within the timezone to be used by this system. '.
+	'Usually choose a "Continent/City". Only choose a special or "Etc" entry if you understand why you need to use it.');
```

Comment:
Yeh, that sounds good. @rbgarga already merged this PR, so @jim-p you can easily make those changes yourself straight on master.